proto/types/params.proto

```diff
@@ -31,7 +31,8 @@ message EvidenceParams {
   option (gogoproto.equal)    = true;
   // Note: must be greater than 0
   int64                    max_age_num_blocks = 1;
-  google.protobuf.Duration max_age_duration   = 2
+  google.protobuf.Duration max_age_duration   = 2;
+  uint32                   max_num_evidence   = 3
```

Comment:
these params are still not used. the proto files in `/proto` can be ignored for the time being.

Reply:
Do you want me to revert the changes on this file then?

Reply:
@marbar3778 ^

Reply:
you can leave it. makes it easier for me later on 😄